Lib/pathlib.py

```diff
@@ -1202,6 +1122,9 @@ def is_mount(self):
         """
         Check if this path is a POSIX mount point
         """
+        if self._flavour is not posixpath:
```

Comment:
You know what I'm going to ask about this line. 😉

Reply:
I'd like to make this:

```
return self._flavour.ismount(self)
```

See also #86943.

However, the current implementation will be useful for `AbstractPath`, so I've been avoiding deleting it. Perhaps I should make the change now, and rescue the old implementation from Git history when we come to add `AbstractPath`?

Reply:
I've marked #31458 as ready for review

Reply:
Well, I just merged #31458, so now you can make the change. 🙂

Reply:
\o/ I've merged in main.